composer.json

```diff
@@ -27,13 +27,13 @@
         "matthiasnoback/symfony-config-test": "^4.0",
         "matthiasnoback/symfony-dependency-injection-test": "^4.0",
         "propel/propel1": "^1.6",
-        "symfony/config": "^3.4 || ^4.3 || ^5.0",
-        "symfony/dependency-injection": "^3.4 || ^4.3 || ^5.0",
-        "symfony/http-foundation": "^3.4 || ^4.3 || ^5.0",
-        "symfony/http-kernel": "^3.4 || ^4.3 || ^5.0",
-        "symfony/phpunit-bridge": "^4.3.8 || ^5.0",
-        "symfony/property-access": "^3.4 || ^4.3 || ^5.0",
-        "symfony/routing": "^3.4 || ^4.3 || ^5.0"
+        "symfony/config": "^4.4 || ^5.0",
+        "symfony/dependency-injection": "^4.4 || ^5.0",
+        "symfony/http-foundation": "^4.4 || ^5.0",
+        "symfony/http-kernel": "^4.4 || ^5.0",
+        "symfony/phpunit-bridge": "^4.4 || ^5.0",
```

Comment:
use 5.1 here

Reply:
Why ?

Reply:
Because we have it on all bundles, it is only for testing purposes, and 5.1 is the consistent value (respect all other sonata bundles)

Reply:
Oh, didn't see it was on require-dev